.github/workflows/ci.yaml

```diff
@@ -81,9 +81,8 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 20
     strategy:
-      fail-fast: true
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9, "3.10"]
+        python-version: [3.6, 3.7, 3.8, 3.9, "3.10", "3.11.0-alpha.7 - 3.11.0"]
```

Comment:
Why use `"3.11.0-alpha.7 - 3.11.0"`? Maybe `3.11-dev` would work?

Reply:
This was just to make sure we would get the latest version (which was necessary for `tomllib` for the `pylint` counter-PR). I'll try with `3.11-dev`.